web/src/styles/scss/activity.scss

```diff
@@ -13,3 +10,8 @@
     color: $color-error-dark;
   }
 }
+
+// Override font weight for heading within cms.gov Review Component
+.entry-details--review div h4.ds-u-font-weight--bold {
+  font-weight: 600 !important;
+}
```

Comment:
I like the idea of adding a comment when using !important.  I'll try to be sure to do this too